### ISSUE TITLE
What is the meaning of "training from scratch" mentioned in paper ?

### ISSUE THREAD

**Issue by susheels** [Shen-Lab/GraphCL#19]
Is it training the downstream svm model with 1) random embeddings or 2) no training GNN embeddings 3) using fully supervised trained GNN embeddings.

I want to know if it is any of the above or something different.

**Comment by yyou1996** [comment]
It is finetuning GNN without pre-training.

**Comment by susheels** [comment]
Thanks for the reply. Just a follow up. 

Q1) So the fine-tuning is with all training labels or 1% or 10%. 

Q2) If it is fine tuning, then is a linear or non linear prediction layer used after GNN.

Thanks

**Comment by yyou1996** [comment]
A1) Yes. Details please see the different setting descriptions.

A2) It depends on the previous SOTA pipelines we followed. Also we refer you to the experiment setting description part, including which SOTA setting we are based on.

A3) SVM is only used in unsupervised learning, following InfoGraph. I would totally agree with your point that it is a stronger classifier than the linear one, while for fair comparison and competitive performance, here we would keep it the same as that in the previous SOTA.

Hope my answer help, thanks!
